dashboard/models.py

```diff
@@ -165,6 +165,8 @@ class Course(models.Model):
     name = models.CharField(max_length=255, verbose_name="Name")
     date_start = models.DateTimeField(verbose_name="Start Date and Time", null=True, blank=True)
     date_end = models.DateTimeField(verbose_name="End Date and Time", null=True, blank=True)
+    ab_test_course = models.BooleanField(blank=False, null=False, default=False, verbose_name=
+                                         "AB test course (For showing grades count on bars)")
 
```

Comment:
I have a couple thoughts on the `ab_test_course` field. Could we name it something that isn't about the AB test? It seems like that is a temporary description of what the field does; I'd prefer something like `show_grade_counts`.

Also, how would you feel about moving this field to the `CourseViewOption` model? It seems like this is more about how a MyLA course is presented (or viewed) than about the course itself. Thoughts?

Reply:
I think @jennlove-um also asked it to be like that but I felt if other universities are looking at it from admin felt won't make sense to then why it is placed so I have name as AB test and extra description what it is.
`CourseViewOption` is more for enabling or disabling the views. It's at higher level. I think We might create separate table saying `Grade_distribution_feature_testing` and have columns like `show_grade_count`, 'Show_y-axis' etc. 
But now I will change it to `show_grade_counts` since you are the second person to ask I should listen

Reply:
I have changed it to `show_grade_counts`